src/main/java/com/epam/training/model/Toy.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.training.model;
+
+import java.util.Objects;
+
+public class Toy extends AbstractChristmasGift {
```

Comment:
Toy is not a Candy, AbstractChristmasGift  is too abstract, they have nothing is common in real world, I would not join them in one hierarchy (and in homework requirements there was nothing about toys)

Reply:
@FilippStankevichTR Done. I removed all non-candy classes.